config/spryk/templates/Glue/Test/GlueTestFixtures/TestFixtures.php.twig

```diff
@@ -46,11 +46,11 @@ class {{ className }} implements FixturesBuilderInterface, FixturesContainerInte
     }
 
     /**
-    * @param {{ resource | normalizeResourceName | ucfirst }}{{ applicationType }}ApiTester $I
+    * @param {{ resource | normalizeResourceName | resourceNameToModelName }}{{ applicationType }}ApiTester $I
     *
     * @return \Generated\Shared\Transfer\{{ zedDomainEntity }}Transfer
     */
-    protected function build{{ zedDomainEntity }}TransferFixture({{ resource | normalizeResourceName | ucfirst }}{{ applicationType }}ApiTester $I): {{ zedDomainEntity }}Transfer
+    protected function build{{ zedDomainEntity }}TransferFixture({{ resource | normalizeResourceName | resourceNameToModelName }}{{ applicationType }}ApiTester $I): {{ zedDomainEntity }}Transfer
```

Comment:
Does the `\SprykerSdk\Spryk\Model\Spryk\Filter\NormalizeResourceNameFilter` still make sense?

Wouldn't 
```suggestion
    protected function build{{ zedDomainEntity }}TransferFixture({{ resource | resourceNameToModelName }}{{ applicationType }}ApiTester $I): {{ zedDomainEntity }}Transfer
```
be easier to understand?